src/functionalTest/resources/application-functional.yaml

```diff
@@ -9,4 +9,4 @@ oauth2.client.secret: ${CA_REF_OAUTH2_CLIENT_SECRET:}
 idam.auth.clientId: ${CA_REF_OAUTH2_CLIENT_ID:}
 idam.auth.redirectUrl: ${OAUTH2_REDIRECT_URI:https://rd-caseworker-ref-api-aat.service.core-compute-aat.internal/oauth2redirect}
 s2s-secret: ${S2S_SECRET:}
-userProfUrl: http://rd-user-profile-api-aat.service.core-compute-aat.internal
\ No newline at end of file
+userProfUrl: http://rd-user-profile-api-pr-664.service.core-compute-preview.internal
```

Comment:
we are pointing to PR

Reply:
Please see my comment here: https://github.com/hmcts/rd-caseworker-ref-api/pull/291#discussion_r617643863

We are pointing to PR so that it can use the Delete API in UP which has not yet gone to Master (still in PR)